### ISSUE TITLE
Classification loss - binary_crossentropy (possibly wrong parameters order)

### ISSUE THREAD

**Issue by zcikojevic** [liuwei16/CSP#87]
https://github.com/liuwei16/CSP/blob/785bc4c5f956860116d8d51754fd76202afe4bcb/keras_csp/losses.py#L12

This is the signature of binary_crossentropy function:
```python
tf.keras.backend.binary_crossentropy(
    target,
    output,
    from_logits=False
)
```
Should the order of input parameters be reversed, like so:
```python
classification_loss = K.binary_crossentropy(y_true[:, :, :, 2], y_pred[:, :, :, 0]) 
```

**Comment by zcikojevic** [comment]
My mistake, didn't check parameters order for the Keras version authors specified.
